assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -734,6 +757,15 @@ describe( 'modules/analytics settings', () => {
 							.getErrorForAction( 'submitChanges' )
 					).toEqual( error );
 					expect( console ).toHaveErrored();
+
+					// getPropertyCreateTime resolver should invoke getProperty on settings change.
```

Comment:
It was invoked because on select property property time would be reset and then on submit changes, resolver would be invoked to get the new property create time. But since it is changed now, to be reset only on `selectProperty` and not called on submit changes, it is not needed here anymore. I removed it